mne/io/bti/bti.py

```diff
@@ -893,7 +893,8 @@ def _read_bti_header_pdf(pdf_fname):
 def _read_bti_header(pdf_fname, config_fname, sort_by_ch_name=True):
     """ Read bti PDF header
     """
-    info = _read_bti_header_pdf(pdf_fname) if pdf_fname else dict()
+    info = _read_bti_header_pdf(pdf_fname) if pdf_fname else \
+        dict({'sample_period': 0.001})
```

Comment:
Had to resort to ugly hack to make the tests pass. Any change we could remove the possibility to have ``sfreq = None``? Ping @dengemann